.circleci/config.yml

```diff
@@ -58,12 +66,32 @@ jobs:
       - setup
       - run: pip install .
       - run: tern -l report -i photon:3.0
+  # Deploy to PyPi
+  pypi_deploy:
+   executor: ubuntu1604
+   steps:
+      - setup
+      - run: pip install .
+      - run: pip install twine
+      # make sure git tag matches release version
+      - run: python setup.py verify
+      - create_pypirc 
+      - run: python setup.py sdist
+      - run: twine upload dist/* 
+
 workflows:
-  # run a full functional test for photonOS
   version: 2
   PRs:
     jobs:
       - linting
       - commit_check
       - security
       - test_changes
+      - pypi_deploy:
```

Comment:
I feel like this should come under a different workflow called 'release'. I think there's something in CircleCI called 'trigger' that would trigger this workflow only if a tag appears.

Reply:
I added a new workflow called `Release` and put the `pypi_deploy` under it. I think we *could* use trigger but don't necessarily need to. The 'filters' I have around `pypi_deploy` will only have it run on version tags.